content/legal-documents/sla.md

```diff
@@ -185,11 +185,22 @@ Payment is due once during a Service Period and the Customer will be charged for
 
 ## 16. Service Level Objectives
 
+Managed OpenShift clusters has this SLA:
+
 | Covered Service | Monthly Uptime Percentage |
 | --- | --- |
 | Zonal Clusters OpenShift API | 99.00% |
 | Regional Clusters OpenShift API | 99.50% |
 
+Managed services and addons running on the OpenShift clusters has this SLA:
+
+| SLA Type | Commitment |
+| --- | --- |
+| Uptime SLA | Best effort - no strict guarantee |
+| Backup SLA | Daily backups where applicable |
+| Incident Response SLA | See the [Support Tiers](https://support.stakater.com/index.html) |
```

Comment:
@karl-johan-grahn do we really talk about incident response times in support sla?